modules/installation-aws-ami-stream-metadata.adoc

```diff
@@ -0,0 +1,31 @@
+//TODO: Add the module include to the following assemblies
+//TODO: Create related modules for OpenStack (QCOW2 and Bare Metal ISO)
+
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-user-infra.adoc
+// * installing/installing_aws/installing-restricted-networks-aws.adoc
+
+[id="installation-aws-ami-stream-metadata_{context}"]
+= Printing {op-system} AMIs for AWS zones by using stream metadata
```

Comment:
Alternate title: "Accessing RHCOS AMIs with stream metadata"
Suggestions for better wording welcome!